test/specs/components/Form/FormField-test.tsx

```diff
@@ -1,28 +1,37 @@
 import * as React from 'react'
-import { isConformant } from 'test/specs/commonTests'
+import { ComponentClass } from 'enzyme'
 
-import { implementsShorthandProp } from '../../commonTests'
-import { Button, RadioGroup, Input, Text, FormField } from '../../../../src/'
+import { isConformant, implementsShorthandProp } from 'test/specs/commonTests'
 import { mountWithProvider } from 'test/utils'
-import Slot from '../../../../src/components/Slot/Slot'
+import { Button, RadioGroup, Input, Text, FormField } from '../../../../src'
+import Slot from 'src/components/Slot/Slot'
```

Comment:
Please align imports, looks a little ugly:

```jsx
import { Button, RadioGroup, Input, Text, FormField } from '../../../../src'
import Slot from 'src/components/Slot/Slot'
```

There should one style:
```
import { Button, RadioGroup, Input, Text, FormField } from 'src'
import Slot from 'src/components/Slot/Slot'
```

Reply:
`src/index` should work

Reply:
done, works thanks